demo/hangman/run.py

```diff
@@ -4,7 +4,7 @@
 secret_word = "gradio"
 
 with gr.Blocks() as demo:    
-    used_letters_var = gr.Variable([])
+    used_letters_var = gr.State([])
```

Comment:
Can we keep one of these demos, using `gr.Variable` so that we can make sure both can be properly used?

Reply:
Hmm good point. Rather than using demos for testing (and showing things that we'll be deprecating), how about making it a test instead?

Reply:
Makes sense to me!

Reply:
Done